.github/ISSUE_TEMPLATE/Proposal.md

```diff
@@ -0,0 +1,9 @@
+---
+name: 🤔 Proposal
```

Comment:
Question: what's the difference between a feature request and a proposal?
Is the proposal more like a discussion?

Reply:
Yes. Also, we can totally throw this out. But this could be one step in the direction of getting closer to EIP style proposals for bigger changes inside Embark. 

E.g. somebody can ask for a feature, but it'd be our job to take that idea and design an API around it that works for everyone. This could be done inside a Proposal.

Does that make sense?

Reply:
Yup, makes sense!